source/Halibut.Tests/Util/TCPListenerWhichKillsNewConnections.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Serilog;
+
+namespace Halibut.Tests.Util
+{
+    public class TCPListenerWhichKillsNewConnections : IDisposable
```

Comment:
I added this since some tests want to start a TCP connection to something that is in someway immediately not working. If the tests was to talk to a random port then it could be that it by chance talks to a port that is actually listening making the test sometimes fail.

This causes the test to be predictable.